fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -126,6 +127,10 @@
      * Default number of records to attempt to run in a single transaction.
      */
     public static final int DEFAULT_LIMIT = 100;
+    /**
+     * Default transaction write size limit. Note that the actual write might be "a little" bigger.
+     */
+    public static final int DEFAULT_WRITE_SIZE = 900 * 1024;
```

Comment:
I suppose 900 kiB is a fine enough default size as any, though I believe that the FDB limits use base-1000 rather than base-1024 units in general (that is to say, the absolute maximum transaction size is 10 MB = 10 * 10<sup>6</sup> B, not 10 MiB = 10 * 2<sup>20</sup> B).

Reply:
900_000 it is.